app/services/proofing/resolution/result_adjudicator.rb

```diff
@@ -37,7 +39,6 @@ def adjudicated_result
               device_profiling_adjudication_reason: device_profiling_reason,
               resolution_adjudication_reason: resolution_reason,
               should_proof_state_id: should_proof_state_id?,
-              double_address_verification: double_address_verification,
```

Comment:
Everywhere else you replaced double_address_verification with ipp_enrollment_in_progress but here you just deleted it.  It looks like this information is removed on analytics.  Is that what we want?